### PR TITLE
新增复制freopen语句功能，优化深色模式错误警告颜色显示

### PR DESCRIPTION
**Pull request by zhouyiqing0304**
## Thank you for your contribution to the XMOJ-Script Community!
哈哈哈不用谢
Please read the comments below to help us consider your Pull Request.

We are all volunteers and completing the process outlined will help us review your commits quicker.

Please use this pull request template, or your pull request will be closed.

**Please make sure you**

1. Base your code and PRs against the repositories developmental branch.

### When you open a pull request, please delete the text above.
---

**What does this PR aim to accomplish?:**

**How does this PR accomplish the above?:**

---
**By submitting this pull request, I confirm the following:**

1. I have read and understood the [contributors guide](https://github.com/XMOJ-Script-dev/XMOJ-Script/blob/dev/CONTRIBUTING.md), as well as this entire template. I understand which branch to base my commits and Pull Requests against.
2. I have commented my proposed changes within the code and I have tested my changes.
3. I am willing to help maintain this change if there are issues with it later.
4. It is compatible with the [GNU General Public License v3.0](https://github.com/XMOJ-Script-dev/XMOJ-Script/blob/dev/LICENSE)
5. I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))
6. I have checked that another pull request for this purpose does not exist.
7. I have considered, and confirmed that this submission will be valuable to others.
8. I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
9. I give this submission freely, and claim no ownership to its content.

---
- [x] I have read the above and my PR is ready for review. *Check this box to confirm*

### CODE REVIEW

**Comment by zhouyiqing0304** [comment]
o

**Comment by zhouyiqing0304** [comment]
我仔细测试过了
开深色模式

**Comment by zhouyiqing0304** [comment]
其实就是在深色模式把颜色换成了黄色

**Comment by zhouyiqing0304** [comment]
https://github.com/XMOJ-Script-dev/XMOJ-Script/issues/501
这个准备一下可以关掉了

**Comment by zhouyiqing0304** [comment]
取消注释的有时间再写

**Comment by zhouyiqing0304** [comment]
顺便加个检测

File: XMOJ.user.js [hunk 1]
```diff
@@ -2633,15 +2633,34 @@ async function main() {
...
                             ErrorElement.style.display = "block";
-                            ErrorMessage.style.color = "red";
+                            if(UtilityEnabled("DarkMode"))ErrorMessage.style.color = "yellow";
+                            else ErrorMessage.style.color = "red";
                             ErrorMessage.innerText = "此题输入输出文件名为" + IOFilename + "，请检查是否填错";
+
+                            let FreopenText = document.createElement('small');
+                            if(UtilityEnabled("DarkMode"))FreopenText.style.color = "white";
+                            else FreopenText.style.color = "black";
+                            FreopenText.textContent = '\n您也可以复制Freopen语句。\n	freopen(\"'+IOFilename+'.in\",\"r\",stdin);\n	freopen(\"'+IOFilename+'.out\",\"w\",stdout);\n';
```
Comment (boomzero) [inline XMOJ.user.js:2643]: freopen

Comment (boomzero) [reply XMOJ.user.js:2643]: Also the variable should be called freopenText, as mentioned in the guidelines

**Review by boomzero — CHANGES_REQUESTED** [review]

**Comment by zhouyiqing0304** [comment]
@boomzero 所以搞了半天你就让我改一个变量名？
okok我去改

**Comment by zhouyiqing0304** [comment]
专业啊，驼峰命名法都出来了

**Comment by zhouyiqing0304** [comment]
好了好了

**Comment by boomzero** [comment]
Okay

**Comment by zhouyiqing0304** [comment]
谢谢

**Pull request merged**